### ISSUE TITLE
Bumped to 4.0.2 - Could not load file or assembly 'Serilog.Sinks.Splunk' or one of its dependencies. Strong name signature could not be verified.

### ISSUE THREAD

**Issue by VictorioBerra** [serilog/serilog-sinks-splunk#188]
```
Could not load file or assembly 'Serilog.Sinks.Splunk' or one of its dependencies. Strong name signature could not be verified.  The assembly may have been tampered with, or it was delay signed but not fully signed with the correct private key. (Exception from HRESULT: 0x80131045)
Description: An unhandled exception occurred during the execution of the current web request. Please review the stack trace for more information about the error and where it originated in the code.

Exception Details: System.IO.FileLoadException: Could not load file or assembly 'Serilog.Sinks.Splunk' or one of its dependencies. Strong name signature could not be verified.  The assembly may have been tampered with, or it was delay signed but not fully signed with the correct private key. (Exception from HRESULT: 0x80131045)
```

![image](https://github.com/serilog-contrib/serilog-sinks-splunk/assets/2934507/65ab9bc6-7460-41d6-8a36-64f48b1b2801)

**Comment by EEParker** [comment]
Is this the root cause since all the builds are on Ubuntu?

https://github.com/serilog-contrib/serilog-sinks-splunk/blob/dev/src/Serilog.Sinks.Splunk/Serilog.Sinks.Splunk.csproj#L13

**Comment by EEParker** [comment]
@VictorioBerra I have verified this merge and build works on my servers, can you verify as well? After that I would like to make a 4.0.3 release.
https://www.nuget.org/packages/Serilog.Sinks.Splunk/4.0.3-dev-00034

**Comment by VictorioBerra** [comment]
Confirmed fixed for me via 4.0.3-dev-00034 @EEParker
